compiler/rustc_mir_transform/src/check_unnecessary_transmutes.rs

```diff
@@ -29,6 +29,7 @@ impl<'a, 'tcx> UnnecessaryTransmuteChecker<'a, 'tcx> {
         function: &Operand<'tcx>,
         arg: String,
         span: Span,
+        r#const: bool,
```

Comment:
pls don't use a raw identifier, it's not easy to read and it doesn't have much value over `in_const_context` or something.